### ISSUE TITLE
Is there a way to drag a point on the X and Y axis to change the data?

### ISSUE THREAD

**Issue by ThuanNIT** [recharts/recharts#3984]
Hi support team recharts 

Now I have a issue . 

How can I drag them like this? 
https://investments.anz.co.nz/kiwisaver-calculator/calculator-start.html 

![DragPoint](https://github.com/recharts/recharts/assets/63941443/b3e0b454-a261-459f-9370-53f7a32956da)

Thanks

**Comment by ckifer** [comment]
This looks like custom behavior that is not supported by recharts. You'll have to use state and dynamically change the values of those lines

**Comment by nikolasrieble** [comment]
I agree that this is not a standard use case. That being said this would be a great addition to storybook, to showcase how recharts can be used / extended to fulfil complex requirements.

@rephaelberkooz Would you like to look into this?
